test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -8,14 +10,15 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
         }
         if ($IsWindows) {
             $userName = "testuserservices"
-            $testPass = "Secret123!"
-            net user $userName $testPass /add > $null
-            $password = ConvertTo-SecureString $testPass -AsPlainText -Force
-            $creds = [pscredential]::new(".\$userName", $password)
+            $testPass = New-CertificatePassword
```

Comment:
@kvprasoon It generate simple password based on New-RandomHexString. To fix tests you have to enhance New-CertificatePassword to generate complex password.

Reply:
how about a new function

```
function New-ComplexPassword {    
    param(        
        [int]$Length            
    )
    $ASCII = 48..90 | Where-Object -FilterScript { $true} | ForEach-Object -Process { [char][byte]$_ }
    For ($Loop = 1; $Loop -le $Length; $Loop++) {
        
        $RandomPassword += ($ASCII | Get-Random)
        
    }    
$RandomPassword -as [string]
}
``

Reply:
You could grub a code from my PR (it creates passwords which works with the CI)
https://github.com/PowerShell/PowerShell/blob/34910f34734b1812c3106a202eff1e24215da107/test/powershell/Modules/Microsoft.PowerShell.AccountManagement/AccountManagement.User.Tests.ps1#L12-L13

Reply:
Thanks @iSazonov - CI is now green